tasks/pkg-windows.yml

```diff
@@ -4,6 +4,12 @@
     msg: "The Datadog ansible role does not currently support Agent 5"
   when: datadog_agent5
 
+- name: Run 6.14.0/1 PowerShell fix
+  win_shell: |
+    Set-ExecutionPolicy Bypass -Scope Process -Force
+    ((New-Object System.Net.WebClient).DownloadFile('https://s3.amazonaws.com/ddagent-windows-stable/scripts/fix_6_14.ps1', $env:temp + '\fix_6_14.ps1'))
```

Comment:
I think we should allow customizing the URL we download the script from, similarly to how we allow customizing the download URL of the Agent MSIs, so that customers that use a mirror of our S3 bucket can also pull the script from their mirror